client-datastore-cassandra/src/main/resources/schema-cassandra.cql

```diff
@@ -26,7 +26,8 @@ CREATE TABLE IF NOT EXISTS tip_wlan_keyspace.client_by_mac_string (
     PRIMARY KEY (customerId, macAddressString)
 ) WITH comment='Index Table to look up wireless client details by macAddress in text form used by TIP WLAN CloudSDK';
 
-CREATE CUSTOM INDEX IF NOT EXISTS idx_client_macAddressString ON tip_wlan_keyspace.client_by_mac_string (macAddressString) USING 'org.apache.cassandra.index.sasi.SASIIndex' WITH OPTIONS = {'mode': 'CONTAINS', 'analyzer_class': 'org.apache.cassandra.index.sasi.analyzer.StandardAnalyzer', 'case_sensitive': 'false'};
+DROP INDEX IF EXISTS idx_client_macAddressString;
+CREATE CUSTOM INDEX IF NOT EXISTS idx_client_macAddressString ON tip_wlan_keyspace.client_by_mac_string (macAddressString) USING 'org.apache.cassandra.index.sasi.SASIIndex' WITH OPTIONS = {'mode': 'CONTAINS', 'analyzer_class': 'org.apache.cassandra.index.sasi.analyzer.NonTokenizingAnalyzer', 'case_sensitive': 'false'};
```

Comment:
this is not good.
every time the ssc-instance is redeployed the index idx_client_macAddressString will be dropped and re-created.
we want it to happen only once - to transition the index from StandardAnalyzer to NonTokenizingAnalyzer.
in order to achieve that the new index has to have a different name from the original one - something like :

DROP INDEX IF EXISTS idx_client_macAddressString;
CREATE CUSTOM INDEX IF NOT EXISTS idx_client_macAddressString_nonTokenizing ON tip_....